crates/egui/src/widgets/color_picker.rs

```diff
@@ -244,15 +244,43 @@ fn color_text_ui(ui: &mut Ui, color: impl Into<Color32>, alpha: Alpha) {
     });
 }
 
-fn color_picker_hsvag_2d(ui: &mut Ui, hsva: &mut HsvaGamma, alpha: Alpha) {
+fn color_picker_hsvag_2d(ui: &mut Ui, hsvag: &mut HsvaGamma, alpha: Alpha) {
+    let srgba_unmultiplied = (std::convert::Into::<Hsva>::into(*hsvag)).to_srgba_unmultiplied();
+    // Send an Opaque Alpha also when Alpha is BlendOrAdditive with negative alpha value (signals Additive blending),
+    // so to hide the alpha's DragValue in both cases.
+    let alpha_control = if alpha == Alpha::Opaque || hsvag.a < 0.0 {
+        Alpha::Opaque
+    } else {
+        alpha
+    };
```

Comment:
This would be equivalent right?
```suggestion
    let alpha_control = if hsvag.a < 0.0 {
        Alpha::Opaque
    } else {
        alpha
    };
```